odfdom/src/main/java/org/odftoolkit/odfdom/doc/table/OdfTableCell.java

```diff
@@ -1126,6 +1127,10 @@ public LocalTime getLocalTimeValue() {
     if (getTypeAttr() == OfficeValueTypeAttribute.Value.TIME) {
       String timeStr = mCellElement.getOfficeTimeValueAttribute();
       try {
+        if (timeStr == null) {
```

Comment:
I don't think this is an improvement: the old code would throw an exception, the new code more or less silently returns null. Further down, this may lead to incorrect results. Better throw an exception. If the user wants to handle this leniently, they should catch the exception.
If on the other hand empty cells with time value type should be allowed, we should return an Optional instead. Are empty time cells allowed in ODF? I don't know.